corehq/apps/export/dbaccessors.py

```diff
@@ -0,0 +1,32 @@
+
+
+def get_latest_case_export_schema_id(domain, case_type):
+    from .models import ExportDataSchema
+
+    key = [domain, 'CaseExportDataSchema', case_type]
+    result = ExportDataSchema.get_db().view(
+        'schemas_by_xmlns_or_case_type/view',
+        startkey=key + [{}],
+        endkey=key,
+        include_docs=False,
+        limit=1,
+        reduce=False,
+        descending=True,
```

Comment:
sounds good. it felt funny at the time, for some reason i always think setting the `_id`/`_rev` is a hack to get couch to do what you want. but agree it'd make it much easier to find the docs we want if there was only one, will update